source/develop/release-management/features/storage/wipe-volume-using-blkdiscard.html.md

```diff
@@ -0,0 +1,84 @@
+---
+title: WipeVolumeUsingBlkdiscard
+authors: ishaby
+wiki_title: Features/WipeVolumeUsingBlkdiscard
+feature_name: Wipe volume using blkdiscard
+feature_modules: vdsm
+feature_status: Work in progress for oVirt 4.1
+---
+
+# Wipe volume using blkdiscard
```

Comment:
The "z" flag is an implementation detail.
Should it really be included in the feature's name?